typescript/sdk/src/providers/ProviderType.ts

```diff
@@ -216,6 +234,12 @@ export interface CosmJsWasmTransaction
   transaction: ExecuteInstruction;
 }
 
+export interface ZKSyncTransaction
+  extends TypedTransactionBase<zkSyncTypes.TransactionRequest> {
+  type: ProviderType.ZkSync;
+  transaction: zkSyncTypes.TransactionRequest;
+}
+
```

Comment:
Doesn't it have to ba added to the enum below?

Reply:
i think it should?

Reply:
How did this not cause an issue in a later PR?

Reply:
ethers/zksync-ethers provider are similar enough in where we use them that it would still be fine with the original ethers provider type

would open up something breaking in the future should we do something that is in the zk-ethers but not in regular ethers